mmv1/products/containerattached/api.yaml

```diff
@@ -0,0 +1,273 @@
+        name: workloadIdentityConfig
+        description: |
+          Workload Identity settings.
+        output: true
+        properties:
+          - !ruby/object:Api::Type::String
+            name: identityProvider
+            description: |
+              The ID of the OIDC Identity Provider (IdP) associated to
+              the Workload Identity Pool.
+          - !ruby/object:Api::Type::String
+            name: issuerUri
+            description: |
+              The OIDC issuer URL for this cluster.
+          - !ruby/object:Api::Type::String
+            name: workloadPool
+            description: |
+              The Workload Identity Pool associated to the cluster.
+      - !ruby/object:Api::Type::NestedObject
+        name: loggingConfig
+        description: |
+          Logging configuration.
+        send_empty_value: true
+        allow_empty_object: true
+        properties:
+          - !ruby/object:Api::Type::NestedObject
+            name: componentConfig
+            description: |
+              The configuration of the logging components
+            send_empty_value: true
+            allow_empty_object: true
+            properties:
+            - !ruby/object:Api::Type::Array
+              name: enableComponents
+              description: |
+                The components to be enabled.
+              send_empty_value: true
+              allow_empty_object: true
+              item_type: !ruby/object:Api::Type::Enum
+                name: 'component'
+                description: |
+                  The components of the logging configuration.
+                values:
+                  - :SYSTEM_COMPONENTS
+                  - :WORKLOADS
+      - !ruby/object:Api::Type::NestedObject
+        name: errors
+        description: |
+          A set of errors found in the cluster.
+        output: true
+        properties:
+          - !ruby/object:Api::Type::Array
+            name: message
+            description: |
+              Human-friendly description of the error.
+            item_type: Api::Type::String
+      - !ruby/object:Api::Type::NestedObject
+        name: authorization
+        description: |
+          Configuration related to the cluster RBAC settings.
+        properties:
+          - !ruby/object:Api::Type::Array
+            name: adminUsers
+            description: |
+              Users that can perform operations as a cluster admin. A managed
+              ClusterRoleBinding will be created to grant the `cluster-admin` ClusterRole
+              to the users. Up to ten admin users can be provided.
+
+              For more info on RBAC, see
+              https://kubernetes.io/docs/reference/access-authn-authz/rbac/#user-facing-roles
+            item_type: Api::Type::String
+      - !ruby/object:Api::Type::NestedObject
+        name: monitoringConfig
+        description: |
+          Monitoring configuration.
+        allow_empty_object: true
```

Comment:
Just want to double-check (since monitoringConfig and managedPrometheusConfig are both optional) that these all actually need to be allow_empty_object & send_empty_value? Although we can support that it's generally something we prefer not to do unless it adds value over just not sending the fields at all.